client/requestUtil.js

```diff
@@ -224,9 +230,9 @@ RequestUtil.prototype.shouldListObject = function (startAt, category) {
   let startAtToCheck = this.normalizeTimestampToMs(startAt, currentTime)
 
   return !startAtToCheck ||
-      (currentTime - startAtToCheck) > parseInt(s3Helper.SQS_RETENTION, 10) * 1000 ||
-      category !== proto.categories.BOOKMARKS ||
-      this.listInProgress === true
+       (currentTime - startAtToCheck) > parseInt(s3Helper.SQS_RETENTION, 10) * 1000 ||
```

Comment:
nit: weird indention

Reply:
fixed that in https://github.com/brave/sync/pull/331/files#diff-93951cfb12e03132f0bd5b9a600a0a27R234